be/src/runtime/runtime_state.h

```diff
@@ -392,9 +392,13 @@ class RuntimeState {
         if (_query_options.__isset.fragment_transmission_compression_codec) {
```

Comment:
should cache the value to avoid check the string value too many times...

Reply:
one query only call once time